cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/CardShowTakenPictureViewImagesAdapter.java

```diff
@@ -141,15 +176,32 @@ public int getItemCount() {
         void updateView(CardShowTakenImage cardShowTakenImage) {
             this.mServiceInspectionsFormFilledRecycleItemBinding.setCardStateEnum(mView.getActualCardState());
 
-            setImageBitmapToImageView(this.mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureViewGeneralCircularImageView, cardShowTakenImage, 8);
+            setImageBitmapToImageView(this.mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureViewGeneralCircularImageView,
+                    cardShowTakenImage, 8);
 
             this.mServiceInspectionsFormFilledRecycleItemBinding.setCardShowTakenImage(cardShowTakenImage);
             this.mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureViewGeneralCircularImageView
                     .setOnClickListener(
                             v -> mView.showPreviewPicDialog(cardShowTakenImage)
-            );
+                    );
             this.mServiceInspectionsFormFilledRecycleItemBinding.executePendingBindings();
         }
+
+        @Override
+        public void onItemSelected() {
+            VibratorUtils.vibrate(mView.getContext(), 400);
```

Comment:
create a variable to describe "400" constant and the programmer not need see method declaration

Reply:
Created